e2e/js_image_docker/WORKSPACE

```diff
@@ -28,11 +28,10 @@ load("@npm//:repositories.bzl", "npm_repositories")
 
 npm_repositories()
 
-# TODO: remove when rules_?  upgrades to modern version supporting
 http_archive(
     name = "bazel_gazelle",
```

Comment:
Why is this needed at all? I guess you don't have to answer or solve that in this PR though 😅

Reply:
Looked up git blame on that comment which confused me here. It came in on a renovate PR. Context here: https://github.com/aspect-build/rules_js/pull/1002/files#r1189105423.

Reply:
Added a detailed comment instead of the TODO. If it is a transitive of rules_docker that is the issue then its never going to be resolved since rules_docker is now archived.